ibodata/lateral_profile.py

```diff
@@ -1,6 +1,63 @@
+import numpy as np
+import math
+
 from beprof.profile import Profile
 
 
 class LateralProfile(Profile):
-    """TODO"""
-    pass
+    def penumbra_left(self):
```

Comment:
Add a docstring explaining what this method returns: 
 - in case of good data
 - in case of corrupted data (i.e. only half of the profile is recorded)